### DISCUSSION TITLE
How to get Coordinates?

### DISCUSSION THREAD

**Discussion by JokerGermany** [Hypfer/Valetudo#1688] [Q&A / Support] [answered]
I want to use the goto Feature in MQTT.
After cleaning the roboter shall go to the bin.
But i don't know how to get the coordinates to use goto...

**Comment by Hypfer** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/974410/212386693-f6941a0c-995e-41b7-aade-2647b40634ee.png)
